docs/development/setup-recommended.md

```diff
@@ -317,7 +317,22 @@ cd zulip
 vagrant up --provider=docker
 ```
 
-The first time you run this command it will take some time because vagrant
+If you are using macOS, you may see something like this:
+```bash
+The provider 'docker' that was requested to back the machine
+'default' is reporting that it is not usable on this system.
+```
+This likely means that Docker is not currently running. You can manually open Docker before starting the development environment: 
+
+```bash
+# On macOS or Linux:
+open a- Docker
+cd zulip
+vagrant up --provider=docker
+```
+Alternatively, you can configure your [Docker settings](https://docs.docker.com/desktop/settings/mac/) to facilitate a quicker launch. 
```

Comment:
Are there clearly enumerable steps we intend a user to follow here? If so, let's add them here to guide users through the rather complex tree of menus in Docker for Mac's settings panes.